AdminLogin.cs

```diff
@@ -16,12 +18,45 @@ public AdminLogin()
         {
             InitializeComponent();
         }
+        
+        public void VerifyUserLogin()
+        {
+            string _userName = userNameTxt.Text.Trim();
+            string _password = txtPassword.Text.Trim();
+
+            if (string.IsNullOrEmpty(userNameTxt.Text))
+                throw new ArgumentException("User Name: ");
+
+            if (string.IsNullOrEmpty(txtPassword.Text))
+                throw new ArgumentException("Password: ");
+
+            SHA1 sha = new SHA1CryptoServiceProvider();
+
+            byte[] bytes  = sha.ComputeHash(Encoding.UTF8.GetBytes(_password));
+
+            StringBuilder builder = new StringBuilder();
+            foreach(var item in bytes)
+            {
+                builder.Append(item.ToString("x2"));
+            }
+            bool ResultVerification = AdminLoginService.VerificationUser(builder.ToString(), _userName);
```

Comment:
bunu yapmam ne kadar mantıklı oldu bilmiyorum. ama password'ü öylece bırakmaya gönlüm el vermedi :)))

bunu net olarak teyit edebilecek bir kaynak bulamadım fakat deneme yanılma yöntemiyle söyle bir sonuca ulaştım. zaten algoritmaya girdi aynı olacağı için çıktı da aynı olacaktır mantık olarak.
 bi password gönderildiğinde oluşan hash  verisini veritabanına kaydettim.
sonra da login yapıldığında oluşan  hash verisiyle veritabanındaki hash edilmiş veriyle karşılaştırdım